### PR TITLE
fix the inline issue in the featured Posts of skeleton ui across all devices

### PR DESCRIPTION
**Pull request by SohamRoy-01**
## Summary

Close the issue [BUG] Featured posts card is not inline with the skeleton card (shimmer card) #109

## Description

fix the inline issue in the featured Posts of skeleton ui across all devices

## Video

https://github.com/krishnaacharyaa/wanderlust/assets/104237973/b9094ec4-75a3-4833-99b0-efe4beacfcf6

### CODE REVIEW

File: frontend/src/components/skeletons/featured-post-card-skeleton.tsx [hunk 1]
```diff
@@ -1,37 +1,36 @@
-import { Skeleton } from '@/components/ui/skeleton';
+import { Skeleton } from "../ui/skeleton";
```
Comment (krishnaacharyaa) [inline frontend/src/components/skeletons/featured-post-card-skeleton.tsx:2]: Why have we changed this? 
Please don't change the entire code, only the required places to be change, and keep it as minimal as possible the changes. For some reason I am guessing this is taken out from the chatgpt...

And the height of the image and the whole card is not aligned in the desktop mode,
And the description needs to be of higher height is what is seems, Please refresh the page and check if they exactly align?

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Thank you for the PR, Kindly follow contributing guidelines. and Address the review

**Comment by SohamRoy-01** [comment]
Hey @krishnaacharyaa, the project is nearly complete. Apologies for the delay, I've been occupied for the past couple of days. I'll ensure to update the pull request (PR) tomorrow.

**Comment by krishnaacharyaa** [comment]
Thank you for the quick update, please resolve the merge conflicts (It is because in the previous PR the classnames were reordered nothing else)... And just one thing to add here is ... the height of the description when its mobile screen is greater than the actual description height ....

Can you kindly fix these 2, will merge the PR

**Pull request closed without merging**